modules/nf-core/trimgalore/main.nf

```diff
@@ -0,0 +1,75 @@
+process TRIMGALORE {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/trim-galore:0.6.7--hdfd78af_0' :
+        'biocontainers/trim-galore:0.6.7--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(reads)
+
+    output:
+    tuple val(meta), path("*{3prime,5prime,trimmed,val}*.fq.gz"), emit: reads
+    tuple val(meta), path("*report.txt")                        , emit: log     , optional: true
+    tuple val(meta), path("*unpaired*.fq.gz")                   , emit: unpaired, optional: true
+    tuple val(meta), path("*.html")                             , emit: html    , optional: true
+    tuple val(meta), path("*.zip")                              , emit: zip     , optional: true
+    path "versions.yml"                                         , emit: versions
```

Comment:
Try to be consistent with the use of parentheses while defining your input paths